### ISSUE TITLE
App container crashing using full + compose examples

### ISSUE THREAD

**Issue by mikecai** [nextcloud/docker#847]
Hi, I'm trying to setup my NextCloud using the docker compose example for nginx-proxy/mariadb-cron-redis/fpm.  For the NextCloud image, I'm building my own using the full/fpm-alpine example Dockerfile, because I would like to install LibreOffice.  However, this combination is causing the NextCloud app container to enter a crash loop when I execute `docker-compose up -d`.  The only change I made was to the docker-compose.yml to use my image for web and cron + setting the environment vars.  If I leave the docker-compose.yml as is where it is using the nextcloud:fpm-alpine image, it runs fine.  I've been at this for couple of hours now, any suggestions on what I maybe doing incorrectly would be really appreciated.  Here's what I see in the docker logs for the app container.

```
docker logs -ft nextcloud_app_1 
2019-08-28T01:53:55.609183422Z Configuring Redis as session handler
2019-08-28T01:53:55.839298076Z /usr/lib/python2.7/site-packages/supervisor/options.py:461: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
2019-08-28T01:53:55.839328773Z   'Supervisord is running as root and it is searching '
2019-08-28T01:53:56.851836938Z Configuring Redis as session handler
2019-08-28T01:53:57.096103131Z /usr/lib/python2.7/site-packages/supervisor/options.py:461: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
2019-08-28T01:53:57.096131614Z   'Supervisord is running as root and it is searching '
2019-08-28T01:53:57.099474899Z Unlinking stale socket /run/supervisord.sock
2019-08-28T01:53:58.260561818Z Configuring Redis as session handler
2019-08-28T01:53:58.492818686Z /usr/lib/python2.7/site-packages/supervisor/options.py:461: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
2019-08-28T01:53:58.492849843Z   'Supervisord is running as root and it is searching '
2019-08-28T01:53:58.496005263Z Unlinking stale socket /run/supervisord.sock
2019-08-28T01:53:59.878067813Z Configuring Redis as session handler
2019-08-28T01:54:00.113448900Z /usr/lib/python2.7/site-packages/supervisor/options.py:461: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
2019-08-28T01:54:00.113478285Z   'Supervisord is running as root and it is searching '
2019-08-28T01:54:00.116836401Z Unlinking stale socket /run/supervisord.sock
...
...
```

**Comment by J0WI** [comment]
PRs are welcomed :wink:

**Comment by joshtrichards** [comment]
Fixed in #925 on 2020-01-09. Closing this out was simply previously overlooked.
